src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-navDropdown/Components/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
+import { DashboardNavigationDropdown } from './index';
+
+import { Icon } from '../styles';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+
+const props = {
+  Options: [{ title: 'title' }],
+  Name: 'name',
+};
+
+it('should render', () => {
+  const tree = render(<DashboardNavigationDropdown {...props} />);
+  expect(tree).toMatchSnapshot();
+});
+
+describe('<DashboardNavigationDropdown />', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(<DashboardNavigationDropdown {...props} />);
+  });
+
+  it('allows us to set Name props', () => {
+    expect(wrapper.props().Name).toEqual('name');
+    wrapper.setProps({ Name: 'different name' });
+    expect(wrapper.props().Name).toEqual('different name');
+  });
+
+  it('allows us to set Options props', () => {
+    expect(wrapper.props().Options).toEqual([{ title: 'title' }]);
+    wrapper.setProps({
+      Options: [{ title: 'second title' }, { title: 'third title' }],
+    });
+    expect(wrapper.props().Options).toEqual([
+      { title: 'second title' },
+      { title: 'third title' },
+    ]);
+  });
```

Comment:
Remove both test cases, instead add

```it should render Name prop inside NavigationHeading surrounded by static icons``` 
- Find total Icon length inside ```NavigationHeading```, it must be 2.
- First icon must be ```StudentIcon``` & second one ```FilledArrowDownIcon```
- Text of ```NavigationHeading``` must be same as provided in prop ```Name``` (you have to trim both icon names before checking)

Complete above test case and let me know if you have any issues/doubts.

```it should render Options only after simulating click```
We will work on this once you are done with first one

Reply:
> - ... (you have to trim both icon names before checking)

Hi @Marvin9, I'm having some trouble understanding this.

Why would checking the **text** of `NavigationHeading` need me to trim both `Icon` names?  To me, I feel that it does not relate to the `Icon` component.

Reply:
@Marvin9 after about 5 minutes I figured out why LOL